at_client_mobile/lib/src/keychain_manager.dart

```diff
@@ -655,4 +658,77 @@ class KeyChainManager {
     }
     return result;
   }
+
+  /// The function write String data to BiometricStorageFile
+  /// If Platform is Windows, data will separated into segments before save. Because in Window, BiometricStorage limit the data length saved
+  Future<void> _writeDataToStore({
+    required BiometricStorageFile store,
+    required String data,
+  }) async {
+    if (Platform.isWindows) {
+      final dataList = _splitString(data, _kWindowSegmentDataLength);
+      await store.write(dataList.length.toString());
+      PackageInfo packageInfo = await PackageInfo.fromPlatform();
+      final packageName = packageInfo.packageName;
+
+      for (int i = 0; i < dataList.length; i++) {
+        final dataStore = await BiometricStorage().getStorage(
+          '${packageName}_data_$i',
+          options: StorageFileInitOptions(
+            authenticationRequired: false,
+          ),
+        );
+        await dataStore.write(dataList[i]);
+      }
+    } else {
+      await store.write(data);
+    }
+  }
+
+  /// The function read String data to BiometricStorageFile
+  Future<String?> _readDataFromStore({
+    required BiometricStorageFile store,
+  }) async {
+    if (Platform.isWindows) {
+      final segmentCount = int.tryParse(await store.read() ?? '0') ?? 0;
+      PackageInfo packageInfo = await PackageInfo.fromPlatform();
+      final packageName = packageInfo.packageName;
+      final results = <String>[];
+      for (int i = 0; i < segmentCount; i++) {
+        final dataStore = await BiometricStorage().getStorage(
+          '${packageName}_data_$i',
+          options: StorageFileInitOptions(
+            authenticationRequired: false,
+          ),
+        );
+        results.add(await dataStore.read() ?? '');
+      }
+      return _combineString(results);
+    } else {
```

Comment:
else block is not needed 
e.g
if( ) {
return _combineString(results);
}
 return  await store.read();

Reply:
I've edited it